sources/packages/web/src/components/generic/ChipTag.vue

```diff
@@ -0,0 +1,22 @@
+<template>
```

Comment:
Even those it's just a simple wrapper around v-chip, I created this a separate component because it's semantically different than ChipLabel and ChipStatus and I wanted to standardize spacing/size/variant for our tags.